html/Task_5/index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Newspaper</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Kaushan+Script&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
+
+  </head>
+  <body>
+    <!-- Website title -->
+    <h1 class="page-title page-title--font page-title--border">
+      The Best Newspaper
+    </h1>
+
+    <!-- Daily informations bar -->
+    <div class="info-bar info-bar--background info-bar--border">
+      <!-- Tempareture -->
+      <span class="info-bar__span info-bar__temperature">
+        <img src="assest/rain-icon.png" alt="Weather-icon" />
+        <b>13<sup>o</sup>C</b></span
+      >
```

Comment:
maybe it will be easier to read? :)

also here you probably need div, as span more for inline elements 
```
<span class="info-bar__span info-bar__temperature">
        <img src="assest/rain-icon.png" alt="Weather-icon" />
        <b>13<sup>o</sup>C</b>
</span>
```